src/main/java/de/gessnerfl/fakesmtp/server/impl/SmtpServerFactoryImpl.java

```diff
@@ -21,10 +22,14 @@ public SmtpServerFactoryImpl(EmailPersister emailPersister, SmtpServerConfigurat
         this.configurator = configurator;
     }
 
+    @Value("${fakesmtp.smtpServer.maxConnections:40}")
+    Integer maxConnections;
```

Comment:
Instead of just using @Value I would add this to the configuration class `FakeSmtpConfigurationProperties`.